src/create_documentation.ts

```diff
@@ -445,7 +445,7 @@ async function copyCssFiles(baseOutDir: string): Promise<string[]> {
 async function copyJavaScriptFiles(baseOutDir: string): Promise<string[]> {
   const scriptOutputDir = path.join(path.resolve(baseOutDir), "scripts");
   const scripts = [
-    path.join(currentDir, "scripts/lunr.js"),
+    path.join(currentDir, "scripts/fuse.js.js"),
```

Comment:
there is an extra ".js"

Reply:
Yeah I think I wanted to be a smart-ass here because the lib is called `fuse.js` but that's just so dumb, fixed.